mne/datasets/hf_sef/hf_sef.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# Authors: Jussi Nurminen <jnu@iki.fi>
+# License: BSD Style.
+
+
+import tarfile
+import os.path as op
+import os
+from ...utils import _fetch_file, verbose
+from ..utils import _get_path, logger, _do_path_update
+
+
+@verbose
+def data_path(set='evoked', path=None, force_update=False, update_path=None,
+              verbose=None):
```

Comment:
Travis is failing because this [needs a docstring](https://travis-ci.org/mne-tools/mne-python/jobs/279898713#L3080).

Also please add `mne.datasets.hf_sef` to the (woefully incomplete) list [here](https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_docstring_parameters.py#L27)